### ISSUE TITLE
Update Beautacus default currency to euros

### ISSUE THREAD

**Issue by RadhepS** [SAP/cloud-commerce-spartacus-storefront#2703]
Change the default currency to EUR to prevent errors being thrown when searching products.

**Comment by RadhepS** [comment]
PR: https://stash.hybris.com/projects/C3PO/repos/beautacus/browse?at=refs%2Fheads%2Ffeature%2FGH-2703

**Comment by RadhepS** [comment]
Closing ticket as it's no longer needed due to USD sample data being available once the ticket below is merged.
https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/2708
